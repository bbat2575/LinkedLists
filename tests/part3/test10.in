NEW 1
1
NEW 2
ha
bazinga
NEW 3
@
bee
c
NEW 7
no
{0}
&
{1}
then
{2}
!
REMOVE 0
REMOVE 1
REMOVE 2
DELETE 3 1
REMOVE 0
DELETE 3 2
REMOVE 1
DELETE 3 -2
REMOVE 2
NEW 1
1
NEW 2
ha
bazinga
NEW 3
@
bee
c
INSERT 3 -4 {4}
INSERT 3 3 {5}
INSERT 3 -2 {6}
REMOVE 4
REMOVE 5
REMOVE 6
DELETE 3 1
REMOVE 4
DELETE 3 2
REMOVE 5
DELETE 3 -2
REMOVE 6
