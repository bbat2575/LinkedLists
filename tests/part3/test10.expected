
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 1

> List 1: ha -> bazinga

> List 2: @ -> bee -> c

> Nested 3: no -> {List 0} -> & -> {List 1} -> then -> {List 2} -> !

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> List 3: no -> & -> {List 1} -> then -> {List 2} -> !

> List 0 has been removed.

Number of lists: 3
List 1
List 2
Nested 3

> List 3: no -> & -> then -> {List 2} -> !

> List 1 has been removed.

Number of lists: 2
List 2
Nested 3

> List 3: no -> & -> then -> !

> List 2 has been removed.

Number of lists: 1
List 3

> List 4: 1

> List 5: ha -> bazinga

> List 6: @ -> bee -> c

> Nested 3: no -> {List 4} -> & -> then -> !

> Nested 3: no -> {List 4} -> & -> {List 5} -> then -> !

> Nested 3: no -> {List 4} -> & -> {List 5} -> then -> {List 6} -> !

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> INVALID COMMAND: REMOVE

> List 3: no -> & -> {List 5} -> then -> {List 6} -> !

> List 4 has been removed.

Number of lists: 3
Nested 3
List 5
List 6

> List 3: no -> & -> then -> {List 6} -> !

> List 5 has been removed.

Number of lists: 2
Nested 3
List 6

> List 3: no -> & -> then -> !

> List 6 has been removed.

Number of lists: 1
List 3

> 
